src/wovnio/wovnphp/Headers.php

```diff
@@ -313,19 +313,44 @@ public function removeLang($uri, $lang = null)
 
         $lang_code = $this->store->convertToCustomLangCode($lang);
         $default_lang = $this->store->settings['default_lang'];
-        if ($this->store->hasDefaultLangAlias()) {
+        if ($lang_code && $this->store->hasDefaultLangAlias()) {
```

Comment:
This part is a little awkward just because of the way custom lang aliases is implemented. We "remove lang code" followed by "add default lang code".

The goal of `removeLang` is to change `/fr/page.php` into `/default_alias/page.php`. It does not need to do anything when given `/page.php` (where `$lang_code` is empty string).

The else block calls `Url::removeLangCode` which exits early if the lang code provided is empty. This isn't clean but I won't change it here.